app/templates/src/test/resources/config/_application.yml

```diff
@@ -11,6 +11,9 @@ async:
     maxPoolSize: 50
     queueCapacity: 10000
 
+liquibase:
+    context: test
```

Comment:
Oh sorry I didn't see the "test" directory -> you are correct, don't change it!!